server/php/README.md

```diff
@@ -0,0 +1,63 @@
+# Stripe Payments Demo — PHP Server
+
+This demo uses:
+- [Slim](http://www.slimframework.com/) as the API framework.
+- [stripe-php](https://github.com/stripe/stripe-php) as the SDK to interact with Stripe's APIs
+- [monolog/monolog](https://github.com/Seldaek/monolog) as the logging interface
+
+## Payments Integration
+
+- [`settings.php`](settings.php) contains the Stripe account configuration as well as the payment methods accepted in this demo.
+- [`index.php`](index.php) contains the routes that interface with Stripe to create PaymentIntents and receive webhook events.
+- [`store`](store) contains inventory and shipping utils.
+
+## Requirements
+
+You’ll need the following:
+
+- PHP 5.4 or higher
+- Modern browser that supports ES6 (Chrome to see the Payment Request, and Safari to see Apple Pay).
+- Stripe account to accept payments ([sign up](https://dashboard.stripe.com/register) for free!)
+
+## Getting Started
+
+After cloning the repository, you first have to install the dependencies using composer:
+
+```
+cd server/php
+composer install
+```
+
+Then update your `settings.php` file with your own [Stripe API keys](https://dashboard.stripe.com/account/apikeys) and any other configuration details you might want to add.
+
+That's it, you can now run the application by using the PHP built-in server still from the `server/php` directory:
+
+```
+php -S localhost:8888 -t ./ index.php
+```
+
+You should now see it running on [`http://localhost:8888/`](http://localhost:8888/)
+
+Optionally you can change the directory tree to fit your needs, don't forget to:
+- change the path to the `public` directory in `settings.php`
+- uncomment the `RewriteBase` option in [`.htaccess`](.htaccess) and update the value accordingly if needed
+
+### Testing Webhooks
+
+If you want to test [receiving webhooks](https://stripe.com/docs/webhooks), we recommend using ngrok to expose your local server.
+
+First [download ngrok](https://ngrok.com) and start your PHP application.
+
+[Run ngrok](https://ngrok.com/docs). Assuming your PHP application is running on the port 8888, you can simply run ngrok in your Terminal in the directory where you downloaded ngrok:
+
+```
+./ngrok http 8888
```

Comment:
should remove `./` since ngrok is installed globally

Reply:
Done!